pkg/account/user.go

```diff
@@ -58,9 +58,11 @@ func (s *server) CurrentUser(w http.ResponseWriter, r *http.Request) {
 	webhandlers.JSON(w, r, struct {
 		User       json.RawMessage `json:"user"`
 		LoggedInAt *time.Time      `json:"logged_in_at"`
+		ExpiresAt  *time.Time      `json:"expires_at"`
```

Comment:
1. Shouldn't this be `SessionExpiresAt` /  `session_expires_at`?
2. While we're at it, we may also want to render the `session.SessionId`.